examples/Advanced-Building-your-own-architecture.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>Can we explicit explain the <code>.connect</code> function? Which blocks do support them? All? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>It's a method supported by any Block. I added a section <code>Connects Methods</code> that explain this method.  </p>